### PR TITLE
Release changes

### PR DESCRIPTION
**Pull request by jala-dx**

### CODE REVIEW

File: src/markdown-pages/add-ons/ekco.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ The EKCO operator is responsible for appending nodes to the CephCluster `storage
 
 EKCO is also responsible for adjusting the Ceph block pool, filesystem, and object store replication factor up and down in accordance with the size of the cluster from `min_ceph_pool_replication` (**Default:** 1) to `max_ceph_pool_replication` (**Default:** 3).
 
+In order to allow for single node Rook clusters, the kURL install script will ease the pod anti-affinity rules for the Rook MDS daemons.
+Once the cluster is scaled beyond one node, EKCO will revert this change in anti-affinity and rebalance the MDS pods.
+This functionality can be disabled by setting the `ekco.rookShouldDisableReconcileMDSPlacement` property to `true`.
+
```
Comment (paigecalvert) [inline src/markdown-pages/add-ons/ekco.md:99]: I'm not able to commit this change directly, but recommended re-write on this is:

```
To allow for single node  Rook clusters, the kURL install script reduces the Pod anti-affinity rules for the Rook MDS daemons.
When the cluster is scaled larger than one node, EKCO reverts this reduction in anti-affinity rules and rebalances the MDS pods.
You can disable this functionality by setting the `ekco.rookShouldDisableReconcileMDSPlacement` property to `true`.
```

**Review by paigecalvert — APPROVED** [review]
Just one recommended re-write!

File: src/markdown-pages/add-ons/ekco.md (hunk 1 above)
Comment (emosbaugh) [reply src/markdown-pages/add-ons/ekco.md:99]: it does not reduce it eases

Comment (emosbaugh) [reply src/markdown-pages/add-ons/ekco.md:99]: makes them less strict... relaxes?

Comment (paigecalvert) [reply src/markdown-pages/add-ons/ekco.md:99]: Ah ok, thank you. Yes I think "makes them less strict" would be the right way to say it

**Review by paigecalvert — APPROVED** [review]

**Pull request merged**